rdflib/namespace/__init__.py

```diff
@@ -418,11 +418,14 @@ def __init__(self, graph: "Graph", bind_namespaces: "_NamespaceSetString" = "cor
             for prefix, ns in _NAMESPACE_PREFIXES_CORE.items():
                 self.bind(prefix, ns)
         elif bind_namespaces == "cc":
```

Comment:
Why use the TLD name for `prefix.cc`? `cc` is also the conventional prefix for [creativecommons](http://creativecommons.org/ns#), so this might be confusing.

(I would also prefer having this as a utility (e.g. `graph.bind_namespaces(util.get_prefix_cc()`) rather than a flag to Graph. That would be more explicit and lets the user control network access and caching.)

Reply:
@niklasl I am happy to do whatever the RDFLib team decides on, but this interface and nomenclature was already predefined, so I just filled it in with an implementation as suggested.

I agree that since prefix.cc relies on a network connection that this is a bit of a tricky situation.

Reply:
Actually,  you can achieve the same effect using RDFLib as is, using just:
```python
from rdflib import Graph

graph = Graph()
graph.parse('https://prefix.cc/context.jsonld')

for pfx, ns in graph.namespaces():
    print(pfx, ns)
```
This yields one very interesting difference though: the `go` prefix won't work, as it ends in a `_`, which is not [treated as a namespace prefix in JSON-LD 1.1](https://www.w3.org/TR/json-ld/#compact-iris), since it does not en with a [URI gen-delim](https://www.rfc-editor.org/rfc/rfc3986#section-2.2) (it has to be explicitly declared using `"@prefix": true` in the context to be treated as a prefix anyway).